tests/test_wrappers.py

```diff
@@ -1224,24 +1223,6 @@ def testpostprocess_y(self):
         assert len(wrappers.BaseWrapper.postprocess_y(y)[1]) == 0
 
 
-class TestUnsetParameter:
-    """Tests for appropriate error on unfitted models.
-    """
-
-    @pytest.mark.filterwarnings("ignore::FutureWarning")
-    def test_unset_input_parameter(self):
-        class ClassBuildFnClf(wrappers.KerasClassifier):
-            def __init__(self, input_param):
-                # does not set input_param
-                super().__init__()
-
-            def _keras_build_fn(self, hidden_dim):
-                return build_fn_clf(hidden_dim)
-
-        with pytest.raises(RuntimeError):
-            ClassBuildFnClf(input_param=10)
```

Comment:
Removing this check and test. It is really a redundant feature that no other estimators have. It was much more helpful previously when the interface was less defined.